depends/packages/rapidcheck.mk

```diff
@@ -0,0 +1,21 @@
+package=rapidcheck
+
+$(package)_version=1.0
+
+$(package)_download_path=https://github.com/Christewart/rapidcheck/releases/download/1.0
+
+$(package)_file_name=$(package)-$($(package)_version).tar.gz
+
+$(package)_sha256_hash=c228dc21ec24618bfb6afa31d622d1f4ea71168f04ee499e1ffcfc63cd5833f4
```

Comment:
@Christewart Please try to fetch https://github.com/MarcoFalke/bitcoin/commit/befb827ccd12267083921e866b97179ba9899902 (Mf1608-rapidcheckRework) and push it here. This should fix your compile issues for now.

Please note that travis will likely reject this due to a bug in gcc4.8.x: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=55914